src/common/utils/oidc/helper.go

```diff
@@ -362,6 +375,29 @@ func GroupsFromClaims(gp claimsProvider, k string) ([]string, bool) {
 	return res, true
 }
 
+type populate func(groupNames []string) ([]int, error)
+
+func populateGroupsDB(groupNames []string) ([]int, error) {
+	return group.PopulateGroup(models.UserGroupsFromName(groupNames, common.OIDCGroupType))
+}
+
+// InjectGroupsToUser populates the group to DB and inject the group IDs to user model.
+// The third optional parm is for UT only, when using the func, the third
```

Comment:
the third what? no need the specify the thire para besides UT?

Reply:
it's incomplete editing of comment...  I'll correct it.